src/NzbDrone.Core/MediaFiles/MediaFileDeletionService.cs

```diff
@@ -61,8 +61,9 @@ public void DeleteMovieFile(Movie movie, MovieFile movieFile)
 
             if (_diskProvider.GetDirectories(rootFolder).Empty())
             {
-                _logger.Warn("Movie's root folder ({0}) is empty.", rootFolder);
-                throw new NzbDroneClientException(HttpStatusCode.Conflict, "Movie's root folder ({0}) is empty.", rootFolder);
+                const string template = "Movie's root folder ({0}) is empty. Protection mechanism has been triggered incase this is a filesystem mounting issue. Nothing will be deleted.";
```

Comment:
Also no const here so it stays in line with the rest of the code base

Reply:
Done